tests/integration/release/conftest.py

```diff
@@ -91,20 +100,6 @@ def pytest_addoption(parser: pytest.Parser) -> None:
 ############
 # Fixtures #
 ############
-@pytest.fixture(scope="session")
```

Comment:
We moved this away from a fixture to avoid scoping issues that arise from running pytest through a command vs within subprocess